pom.xml

```diff
@@ -100,6 +100,11 @@
       <version>4.11</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.plugin</groupId>
+      <artifactId>hydrator-common</artifactId>
+      <version>2.4.0-SNAPSHOT</version>
```

Comment:
Also lets not use snapshot version in release branch. Make it a property with `2.4.0` version